src/App.tsx

```diff
@@ -15,71 +16,94 @@ export const goods = [
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    return (
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <main className="section container">
+        {selectedGood === ''
+          ? (<h1 className="title">No goods selected</h1>)
+          : (
+            <h1 className="title is-flex is-align-items-center">
+              {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+              { selectedGood } is selected
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => this.setSelectedGood('')}
+              />
+            </h1>
+          )}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        <table className="table">
+          <tbody>
+            {goods.map(good => (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={
+                  classNames(
+                    {
+                      // eslint-disable-next-line max-len
+                      'has-background-success-light': selectedGood.includes(good),
+                    },
+                  )
+                }
+              >
+                <td>
+                  {!selectedGood.includes(good)
+                    ? (
+                      <button
+                        data-cy="AddButton"
+                        type="button"
+                        className="button"
+                        onClick={
+                          () => this.setSelectedGood(good)
+                        }
+                      >
+                        +
+                      </button>
+                    )
+                    : (
+                      <button
+                        data-cy="RemoveButton"
+                        type="button"
+                        className="button is-info"
+                        onClick={
+                          () => this.setSelectedGood('')
+                        }
+                      >
+                        -
+                      </button>
```

Comment:
only `data-cy` and content differ, right? `className` and `onClick` slightly change